mock/mock.go

```diff
@@ -784,7 +784,8 @@ type IsTypeArgument struct {
 // alternative to AnythingOfType.  Used in Diff and Assert.
 //
 // For example:
-// Assert(t, IsType(""), IsType(0))
+//
+//	Assert(t, IsType(""), IsType(0))
```

Comment:
added indentation here too

Reply:
This is irrelevant to the scope of this PR. Please submit this change as a separate PR.